core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/RegionExecutionCoordinator.scala

```diff
@@ -97,13 +96,6 @@ class RegionExecutionCoordinator(
       .flatMap(_ => connectChannels(region.getLinks))
       .flatMap(_ => openOperators(operatorsToInit))
       .flatMap(_ => sendStarts(region))
-      .rescue {
```

Comment:
What is the reason of removing this?

Reply:
Previously, we used this code to notify the client that the workflow could not be scheduled. However, the old code hid the exact operator information. I have now simplified this by throwing the exception and allowing the caller to handle it. If the error contains operator information, the caller can utilize it.